packages/shared_preferences/shared_preferences/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.2
+
+* Updates documentation for `containsKey`
```

Comment:
This is missing a period; please see the CHANGELOG style guide linked from the PR checklist.

Reply:
Sorry :sweat_smile:

I also just queued the changes for NEXT as I don't think a version bump is needed. Correct me if I should change it back.

Reply:
Ok I somehow miss read 
> Non-code changes that we make to future-proof development in some way, but don't directly benefit clients. Examples of this include

And only saw the part with `Non-code changes`. 
I'll revert it again.